gradle.properties

```diff
@@ -28,6 +28,7 @@ reactorExtensionsVersion = 1.1.2
 slf4jVersion = 1.7.30
 springBootVersion = 2.4.2
 springVersion = 5.3.3
+federationGraphQLJavaSupport = 0.6.0
```

Comment:
All other props for version have version suffix -> lets add it for consistency (we can drop the support suffix)